src/lib/components/Tab/index.tsx

```diff
@@ -37,60 +49,110 @@ export const tabItemStyleClasses: Record<TabStyle, Record<TabItemStatus, string>
   },
 };
 
-export type TabsProps = PropsWithChildren<{
-  className?: string;
-  style?: TabStyle;
-}>;
+export type TabsProps = PropsWithChildren<
+  ComponentProps<'div'> & {
+    className?: string;
+    style?: TabStyle;
+  }
+>;
 
-export const TabsComponent: FC<TabsProps> = ({ children, className, style = 'default' }) => {
+export const TabsComponent: FC<TabsProps> = ({ children, className, style = 'default' as TabStyle, ...rest }) => {
+  const idPrefix = nanoid();
```

Comment:
This is gonna be different for every render call, can you please use `useMemo`?

Reply:
Could you demo how that would look here? That's outside my understanding. I understand memoization from learning it in school, but I'm not great at identifying where it can be useful.

Reply:
It will look something like this 
```ts
const idPrefix = useMemo(() => nanoid(), []);
```